File: Java/src/World/body/plant/Guarana.java
package World.body.plant;

import GUI.Color_obj;
import World.Point;
import World.World;
import World.body.Plant;

import java.io.Serializable;

public class Guarana extends Plant implements Serializable {
    private static final long serialVersionUID = -987654321L;
    public Guarana(Point point_location, World world) {
        super(point_location, world);
        setName("Guarana");
        setPower(0);
        setColor(Color_obj.GUARANA);
    }

    @Override
    public int get_stronger() {
        return 3;
    }
}

File: Python/World/Body/Plant/guarana.py
from World.Body.plant import Plant
from World.point import Point
from GUI.colors import Colors


class Guarana(Plant):
    def __init__(self, point_location, world):
        super().__init__(point_location, world)
        self.set_name("Guarana")
        self.set_power(0)
        self.set_color(Colors.GUARANA)

    def get_stronger(self):
        return 3
